### DISCUSSION TITLE
Need new release

### DISCUSSION THREAD

**Discussion by vancluever** [go-acme/lego#2194] [Q&A] [answered]
### Welcome

- [X] Yes, I've searched similar issues on GitHub and didn't find any.

### How do you use lego?

Library

### Detailed Description

Hey @ldez et al, looks like there hasn't been a tagged release in a few months, can you cut one?

We got folks waiting for #2155 in the TF ACME provider.

Thanks!

**Comment by ldez** [comment, marked as answer]
I also had several requests on different subjects, so maybe it's time to create a release.

Note: this version will require go1.22

**Comment by vancluever** [comment]
@ldez I see the release, I'm pulling it in now - thanks! Will close momentarily here when everything looks good.

**Reply by vancluever** [reply]
Looking good and our release is building. Thanks again @ldez!
